README.md

```diff
@@ -51,6 +51,7 @@ trace.Record(Annotations.ServerRecv());
 trace.Record(Annotations.ServiceName(serviceName));
 trace.Record(Annotations.Rpc("GET"));
 trace.Record(Annotations.ServerSend());
+trace.Record(Annotations.Tag("http.Get", "<url>")); //adds binary annotation
```

Comment:
we should probably use the correct key name :) "http.url"

https://github.com/openzipkin/zipkin-api/blob/master/thrift/zipkinCore.thrift#L150